scss/_patterns_tabs.scss

```diff
@@ -77,7 +77,7 @@
       }
 
       // make sure tab strip border doesn't overlap focus outline
-      &:focus {
+      &:focus-visible {
```

Comment:
We still need `focus` fallback for older browsers (as described in other comment).

Reply:
Fixed this one also.